aten/src/ATen/core/PhiloxRNGEngine.h

```diff
@@ -211,17 +210,19 @@ class philox_engine {
   }
 
   inline detail::FLOAT2 normalize_pair_uniform(detail::FLOAT2 in) {
+    // TODO(min-jean-cho) change to Polar method, a more efficient version of Box-Muller method
     // TODO(voz) We use std:: below, and thus need a separate impl for CUDA.
     float u1 = in[0];
+    float u2 = in[1];
 
     constexpr float two_pi = 2.0 * M_PI;
 
     float mag = std::sqrt(-2.0 * std::log(u1));
 
     detail::FLOAT2 ret;
 
-    ret[0] = mag * std::cos(two_pi);
-    ret[1] = mag * std::sin(two_pi);
+    ret[0] = mag * std::cos(two_pi * u2);
+    ret[1] = mag * std::sin(two_pi * u2);
     return ret;
```

Comment:
Only `ret[0]` is used from the caller side, why do we bother to compute two values?

Reply:
I think it was because we need two samples from uniform distribution `u1`, `u2` anyways. But, we don't need to bother computing two normal.